packages/@okta/vuepress-theme-prose/components/News.vue

```diff
@@ -77,15 +77,87 @@
           Blog
         </p>
         <h3 class="news__article-title dont-break-out">
-          Streamline Your Okta Configuration in Angular Apps
+          {{ blogPostTitles[4] || 'Streamline Your Okta Configuration in Angular Apps' }}
         </h3>
         <a
           class="news__article-link dont-break-out"
-          href="https://developer.okta.com/blog/2023/03/07/angular-forroot"
+          :href="blogPostLinks[4] || 'https://developer.okta.com/blog/2023/03/07/angular-forroot'"
         >
           Read the blog post
         </a>
       </article>
     </div>
   </section>
 </template>
+
+<script>
+const axios = require('axios');
+
+export default {
+  name: 'News',
+  data() {
+    return {
+      blogPostTitles: [],
+      blogPostLinks: []
+    }
+  },
+  mounted() {
+    this.getBlogPosts();
+  },
+  methods: {
+    xmlToJs(xmlString) {
+      let xmlDoc = new DOMParser().parseFromString(xmlString, 'text/xml');
+      let result = {};
+
+      function parseNode(node, obj) {
+        if (node.nodeType === Node.TEXT_NODE) {
+          if (node.nodeValue.trim() !== '') {
```

Comment:
nit: No strong preference here but if we decide that `obj['#text']` can be an empty string then we can drop this check.